.github/workflows/unittests.yml

```diff
@@ -0,0 +1,28 @@
+name: Operator Unit tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
```

Comment:
Could we use the same version that's included in DEVELOPER.md -- 1.16.2?

Reply:
sure.